.circleci/config.yml

```diff
@@ -340,12 +340,6 @@ jobs:
       - run:
           name: Build and run schema generator
           command: cargo schema --locked
-      - run:
```

Comment:
Should we remove the integration/test.rs file as well?
I would suggest that (and remove devDeps to keep CI fast)

Reply:
I already removed the integ tests, but forgot about the dev deps. Thanks.